docs: Documentation overhaul for streamlined Cymbal Air (#552)
GoogleCloudPlatform/genai-databases-retrieval-app: docs/run_app.md

# Running the Agentic App Demo

##  Before you begin

1. Make sure you have [setup and initialized your
   Database](../README.md#one-time-database--tool-configuration).

> 1. Make sure you are able to [run Toolbox locally](../README.md#launch-the-toolbox-server-choose-one).

Reviewer: ```suggestion
1. Make sure you are able to [launch Toolbox server](../README.md#launch-the-toolbox-server-choose-one).
```

Author: Thanks for pointing that out!

1. Make sure you have Python 3.11+ installed

## Setting up your Environment

1. Set up [Application Default Credentials](https://cloud.google.com/docs/authentication/application-default-credentials#GAC):

    ```bash
    gcloud auth application-default login
    ```

>     > [!TIP]

Reviewer: The Tip in this page is not rendering correctly. Checkout the preview~

Author: Same as the [comment above](https://github.com/GoogleCloudPlatform/genai-databases-retrieval-app/pull/552#discussion_r2198260923).

# Running the Agentic App Demo

##  Before you begin

> 1. Make sure you've [deployed and running Toolbox

Reviewer: ```suggestion
1. Make sure you're able to [run Toolbox
```
I dont think user have to deploy it? The read me instructions also doesn't seem to include instructions to deploy it on Cloud Run.

   locally](../README.md#toolbox-setup).

> 1. Make sure you've [setup and initialized your

Reviewer: ```suggestion
1. Make sure you have [setup and initialized your
```

   Database](../README.md#database-setup).

1. Make sure you have Python 3.11+ installed

## Setting up your Environment

1. Set up [Application Default Credentials](https://cloud.google.com/docs/authentication/application-default-credentials#GAC):

    ```bash
    gcloud auth application-default login
    ```
    > [!TIP]
    > If you are running into `403` error, check to make sure the service
    > account you are using has the `Cloud Run Invoker` IAM in the retrieval
    > service project.

1. Install the dependencies using `pip`. You may wish to do this in a

>    [venv](https://docs.python.org/3/library/venv.html):

Reviewer: ```suggestion
   virtual environment, e.g. [venv](https://docs.python.org/3/library/venv.html):
```
